README.md

```diff
@@ -53,7 +53,7 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 
 Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
 
-CLIMADA is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, version 3.
+CLIMADA is free software: you can redistribute it and/or modify it under the terms of the GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007 as published by the Free Software Foundation, https://github.com/CLIMADA-project/climada_python/blob/main/LICENSE
```

Comment:
If you change the line above, you should also change the line below and also the link that comes at the very end.

Reply:
Hmmm this change to the Readme was done by @emanuel-schmid in commit b945410 on the develop branch.

Reply:
Ah, this is a PR for the main branch, probably by mistake, right?

Reply:
Yes, thanks, that is the problem! I will redo.